recipe/meta.yaml

```diff
@@ -36,7 +38,17 @@ outputs:
         - LICENSE.txt
       summary: 'A meta-package to select Arrow build variant'
 
-  - name: arrow-cpp
+  # compat output for old mutex-package naming
+  - name: arrow-cpp-proc
```

Comment:
Shouldn't this have a `run` requirement for the new `-proc` package?

Reply:
True